versioned_docs/version-8.7/self-managed/identity/mapping-rules.md

```diff
@@ -1,28 +1,31 @@
 ---
-id: managing-mapping-rules
-title: "Managing mapping rules"
-sidebar_label: "Managing mapping rules"
-description: "Manage mapping rules within Identity to support the assigning of Camunda entities to your users."
+id: mapping-rules
+title: "Mapping rules"
+sidebar_label: "Mapping rules"
+description: "Map your auth data to Camunda-specific data using mapping rules."
 ---
 
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-In this guide, you will learn how to manage mapping rules in Identity and how to control the Camunda entities related to
-them.
-
-:::tip
-To learn more about the concept of mapping rules, see
-the [mapping rules documentation](../../../concepts/mapping-rules.md).
+:::info
+Mapping rules are only available for Camunda 8 Self-Managed with [OIDC-based authentication](/self-managed/identity/configuration/connect-to-an-oidc-provider.md).
 :::
 
+Mapping rules are designed to allow you to map Camunda-specific data to your users by using the claims on your JWT token. You can map two types of data:
```

Comment:
I find the first sentence a bit unclear, especially the "allow you to map Camunda-specific data", that's very generic. How about "Mapping rules allow you to dynamically assign Identity entities to your users based on claims in your JWT tokens. You can assign two types of entities with mapping rules:"?

Reply:
I like that. Added.